### ISSUE TITLE
Add method for each API property

### ISSUE THREAD

**Issue by ayuhsya** [abinashmeher999/pymediawiki#21]
Checklist of all API properties for reference:
- [X] images
- [X] linkshere
- [X] categories
- [x] categoryinfo
- [x] contributors
- [ ] deletedrevisions
- [x] duplicatefiles
- [ ] extlinks
- [x] fileusage
- [ ] imageinfo
- [ ] info
- [ ] iwlinks
- [ ] langlinks
- [x] links
- [ ] pageprops
- [x] redirects
- [ ] revisions
- [ ] stashimageinfo
- [ ] templates
- [ ] transcludedin

**Comment by ayuhsya** [comment]
Can we label this issue with **enhancement** and probably **easy-to-fix** too?

**Comment by abinashmeher999** [comment]
@ayuhsya Glad that you think it is easy to fix and that's a good sign because it is huge. I will just add the enhancement label and help wanted.

**Comment by ayuhsya** [comment]
Yes, this task is huge but it's monotonous. We've to ping every API property using _Postman_ and then study the JSON response which is quite similar in every case. I've dropped this task for now as it requires simple parsing. :thinking:

**Comment by abinashmeher999** [comment]
There is no need to Postman. All the APIs can be tested with the API sandbox as mentioned in the MediaWiki documentation.
> I've dropped this task for now as it requires simple parsing. 

It's not mandatory for you to address the issue just because you created it. There are other issues you can work on.

**Comment by abhigyank** [comment]
I think I can get started working on the first two API properties.

**Comment by abinashmeher999** [comment]
@abhigyank The ones with the check mark have already been covered. You should take up ones that are still left.

**Comment by abhigyank** [comment]
I have covered links and contributors. Working on a couple more.
